### PR TITLE
Bump Package Manager version to 2022.11.4-20

### PR DESCRIPTION
**Pull request by glin**
Update Package Manager to 2022.11.4-20 for another hotfix release.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -43,8 +43,8 @@ services:
       context: ./package-manager
       dockerfile: "Dockerfile.${IMAGE_OS:-ubuntu1804}"
       args:
-        RSPM_VERSION: 2022.11.2-18
-    image: rstudio/rstudio-package-manager:2022.11.0
+        RSPM_VERSION: 2022.11.4-20
+    image: rstudio/rstudio-package-manager:2022.11.4-20
```
Comment (glin) [inline docker-compose.yml:47]: Are these `image` tag changes in docker-compose.yml intended? I updated `RSPM_VERSION` and ran `just update-versions`.

File: docker-compose.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ services:
       dockerfile: "Dockerfile.${IMAGE_OS:-ubuntu1804}"
       args:
         RSW_VERSION: 2022.07.2+576.pro12
-    image: rstudio/rstudio-workbench:2022.07.2
+    image: rstudio/rstudio-workbench:2022.07.2-576.pro12
```
Comment (colearendt) [inline docker-compose.yml:11]:
```suggestion
    image: rstudio/rstudio-workbench:2022.07.2
```

File: docker-compose.yml (hunk 1 above)
Comment (colearendt) [inline docker-compose.yml:47]:
```suggestion
    image: rstudio/rstudio-package-manager:2022.11.4
```

**Review by colearendt — APPROVED** [review]
I think this is a bug in the `update-versions` command. We can fix it out of band.

Images now do not have the build number included in their tag, so the compose image reference should not include that

**Comment by glin** [comment]
@colearendt Thanks, the image tags look right now. There are some build failures in the workbench/r-session preview images, but I assume those are unrelated since the same jobs are failing on main. I'll merge when CI passes.

**Pull request merged**